CHANGELOG.md

```diff
@@ -23,9 +24,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.3.1] - 2022-07-25
 
-### Added
-- Add campus hybrid event support. [#915](https://github.com/rokwire/events-manager/issues/915)
```

Comment:
I may have found the issue with the entry for the "Add campus hybrid event" task. Earlier, it was correctly included under 3.3.1 in the CHANGELOG. Please see the diff commit here (https://github.com/rokwire/events-manager/compare/3.3.0...3.3.1). So, I think you just need to add it back under 3.3.1 and remove it from 3.3.0 in this PR. This entry is also missing in the release notes for 3.3.1 here (https://github.com/rokwire/events-manager/releases/tag/3.3.1). Please make these changes when you get a chance. Thanks.